subjects/java/checkpoints/html-validator/README.md

```diff
@@ -0,0 +1,55 @@
+## HTML Validator
+
+### Instructions
+
+Create a class `HTMLValidator` that provides a method to validate whether a given HTML string is correctly formatted. The method should support a few basic HTML tags (`<html>`, `<body>`, `<div>`, `<p>`, `<b>`, `<i>`, `<h1>`, `<h2>`). If the HTML is valid, the method should return `true`. If the HTML is invalid, the method should return `false`.
```

Comment:
I would add that an empty string is considered valid

Reply:
i guess is something clear it's just like the bracket problem, while you have nothing opened and not closed so it's valid no need to add it.